lib/components/more_details_view.dart

```diff
@@ -0,0 +1,299 @@
+                                    .moreDetailModel.question.mediaUrls.length)
+                        ? TextSpan() // if true, no comma
+                        : TextSpan(
+                            // if false, add comma after link
+                            text: ' , ',
+                            style: TextStyle(
+                              fontWeight: FontWeight.bold,
+                              color: Colors.black,
+                            ),
+                          ),
+                  ],
+                ),
+              ),
+            ),
+          ),
+      ],
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Stack(children: [
+          _profileNameAndIcon(),
+          Positioned(
+            right: -10.0,
+            top: widget.isAnswer ? 0.0 : 35.0,
+            child: Transform.scale(
+              scale: widget.isAnswer ? 1.20 : (widget.isCurrUser ? .75 : 1.30),
+              child: widget.isAnswer
+                  ? (widget.isCurrUser
+                      ? IconButton(
+                          color: Color(0xFF00ADB5),
+                          icon: Icon(Icons.message_outlined),
+                          onPressed: () => null,
+                        )
+                      : Container())
+                  : (widget.isCurrUser
+                      ? FlatButton(
+                          color: Colors.red,
+                          shape: RoundedRectangleBorder(
+                            borderRadius: BorderRadius.circular(20.0),
+                          ),
+                          onPressed: () async {
+                            await CloseQuestionService().archiveQuestion(
+                                widget.moreDetailModel.question.id);
+
+                            showDialog(
+                              context: context,
+                              builder: (context) => AlertDialog(
+                                content: Text(
+                                  'Question was archived',
+                                  style: TextStyle(fontWeight: FontWeight.bold),
+                                  textAlign: TextAlign.center,
+                                ),
+                                actions: <Widget>[
+                                  FlatButton(
+                                      onPressed: () => Navigator.push(
+                                            context,
+                                            MaterialPageRoute(
+                                              builder: (context) => Home(),
+                                            ), //archive question
+                                          ),
+                                      child: Text('Back to Home')),
+                                ],
+                              ),
+                            );
+                          },
+                          child: Text("Archive"),
```

Comment:
Make this Archive font bigger too and consider changing the color to #0E86D4? Try that color out it's a light blue.

Reply:
won't the light blue to be calming for something that will delete a question?